lib/public/components/common/form/inputs/colorInputComponent.js

```diff
@@ -0,0 +1,34 @@
+/**
+ *  @license
+ *  Copyright CERN and copyright holders of ALICE O2. This software is
+ *  distributed under the terms of the GNU General Public License v3 (GPL
+ *  Version 3), copied verbatim in the file "COPYING".
+ *
+ *  See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ *  In applying this license CERN does not waive the privileges and immunities
+ *  granted to it by virtue of its status as an Intergovernmental Organization
+ *  or submit itself to any jurisdiction.
+ */
+
+import { h, iconX } from '/js/src/index.js';
+
+/**
+ * Returns a create button component
+ *
+ * @param {string} color The current color go
+ * @param {function} onInput The function of what must happen on input
+ * @return {Component} The color input element
+ */
+export const colorInputComponent = (color, onInput) => h('.flex-row.items-center.g2', [
+    h('input#color', {
+        type: 'color',
+        value: color,
+        oninput: (e) => onInput(e.target.value ?? null),
+    }),
+    h(
+        '.btn.btn-pill.f7',
+        { disabled: !color, onclick: () => onInput(null) },
```

Comment:
Perhaps a title or suggestion is also added to that `x` buttons refers to.
It is not very intuitive that it `resets` the selection. For me while I had the color palette opened, I thought it would close the color palette.

Reply:
I added a title